firecracker/packer/scripts/create_rootfs_image.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
```

Comment:
Having an introductory comment here indicating that, as written, this script is _not_ intended to be run directly on a developer's workstation.